demos/init-db.php

```diff
@@ -188,6 +196,7 @@ protected function init(): void
 
         $this->hasOne($this->fieldName()->client_country_iso, [
             'model' => [Country::class],
+            'our_field' => $this->fieldName()->client_country_iso,
```

Comment:
When prefixing is used, guessing `our_field` for hasOne doesn't work

code: https://github.com/atk4/data/blob/68aa5efd536820d1d7b92061f4c18786425c77e9/src/Reference/HasOne.php#L157

is it possible to fix that?

Reply:
Looks like it's working now